_posts/intro/2014-03-01-raspberry-pi-ubuntu-20-quick-start.md

```diff
@@ -42,6 +42,8 @@ following steps:
    sudo apt update
    sudo apt upgrade
    </pre>
+   
+   {% include alerts.html message="Ubuntu 20.04 currently ships with an issue where the error ```unable to resolve host ubuntu``` is thrown when running sudo. To resolve this edit the file */etc/hosts* and add ```127.0.0.1 ubuntu``` under the first entry." %}
```

Comment:
This might not be the best address since it is already in use by localhost [1]. I'm currently investigating the use of `127.0.1.1 ubuntu` instead of `127.0.0.1 ubuntu`.

[1] [Debian networking reference](https://www.debian.org/doc/manuals/debian-reference/ch05.en.html#_the_hostname_resolution).